projects/plugins/jetpack/modules/subscriptions/subscribe-modal/class-jetpack-subscribe-modal.php

```diff
@@ -132,14 +132,14 @@ public function get_subscribe_template_content() {
 		// translators: %s is the name of the site.
 		$discover_more_from = sprintf( __( 'Discover more from %s', 'jetpack' ), get_bloginfo( 'name' ) );
 		$continue_reading   = __( 'Continue Reading', 'jetpack' );
-		$subscribe_text     = __( 'Subscribe to the newsletter to keep reading and get access to the full archive.', 'jetpack' );
```

Comment:
I decided to kind-of revert the text change. I just cut some text to achieve my goal of keeping it to one line, rather than changing the text in a fundamental way. 

For the record, I still think we should change this. The current text is still misleading - it is currently more what you'd put on a paywall or on a post that is subscribers-only.

But that's a separate conversation.